### PR TITLE
cmd/free: add nused/nfree field in command free

### PR DESCRIPTION
**Pull request by Donny9**
## Summary
cmd/free: add nused/nfree field in command free.
nused describes the number of allocated chunks.
nfree describes the number of free chunks

Change-Id: I74aa4b1e7394a17c3b117322a4cc24aa52aac3b8
Signed-off-by: Jiuzhu Dong <dongjiuzhu1@xiaomi.com>
## Impact
Using cmd:free to show more memory information.
## Testing
daily test.

### CODE REVIEW

**Comment by btashton** [comment]
@Donny9  this looks fine to me but can you also update the documentation here for these new fields 

https://github.com/apache/incubator-nuttx/blob/master/Documentation/applications/nsh/commands.rst#show-memory-manager-status-free

File: mm/mm_heap/mm_mallinfo.c [hunk 1]
```diff
@@ -92,6 +93,7 @@ int mm_mallinfo(FAR struct mm_heap_s *heap, FAR struct mallinfo *info)
           if ((node->preceding & MM_ALLOC_BIT) != 0)
             {
               DEBUGASSERT(node->size >= SIZEOF_MM_ALLOCNODE);
+              aordblks ++;
```
Comment (yamt) [inline mm/mm_heap/mm_mallinfo.c:96]: "No space separating unary operators"

https://github.com/apache/incubator-nuttx/blob/master/Documentation/contributing/coding_style.rst#operators

Comment (Donny9) [reply mm/mm_heap/mm_mallinfo.c]: > "No space separating unary operators"
> 
> https://github.com/apache/incubator-nuttx/blob/master/Documentation/contributing/coding_style.rst#operators

Done. Thank you.

**Comment by Donny9** [comment]
> @Donny9 this looks fine to me but can you also update the documentation here for these new fields
> 
> https://github.com/apache/incubator-nuttx/blob/master/Documentation/applications/nsh/commands.rst#show-memory-manager-status-free

Done, Thank you.

**Review by btashton — APPROVED** [review]

**Pull request merged**